java/src/hu/holyoil/view/frames/GameFrame.java

```diff
@@ -1,30 +1,74 @@
 package hu.holyoil.view.frames;
 
 import hu.holyoil.view.IViewComponent;
+import hu.holyoil.view.panels.EnvironmentPanel;
+import hu.holyoil.view.panels.InventoryListPanel;
+import hu.holyoil.view.panels.LogPanel;
+import hu.holyoil.view.panels.PlayerListPanel;
 
 import javax.swing.*;
+import java.awt.*;
+import java.util.Arrays;
+import java.util.List;
 
 public class GameFrame extends JFrame implements IViewComponent {
+    private LogPanel logPanel;
+    private EnvironmentPanel environmentPanel;
+    private InventoryListPanel inventoryListPanel;
+    private PlayerListPanel playerListPanel;
+
+    private final List<IViewComponent> panels = Arrays.asList(logPanel, environmentPanel, inventoryListPanel, playerListPanel);
```

Comment:
This `asList` function will run before calling the constructors of the panels, so it will contain only nulls

Reply:
Yes. Is that a problem? We fill those references in InitComponents, and the panels list is used only after InitComponents. I saw, you've already moved the initialization in your PR, so I won't change this.

Reply:
There aren't references in that list only nulls. But yes I moved it in my PR so no prob.